### ISSUE TITLE
Datatable: RowEditor broken if sortBy is enabled

### ISSUE THREAD

**Issue by erickdeoliveiraleal** [primefaces/primefaces#4819]
## 1) Environment
It works in 7.0.1 but not in 7.0.2

## 2) Expected behavior

Click on  <p:rowEditor /> should open options.

## 3) Actual behavior

Nothing happens.

## 4) Steps to reproduce

Click on button to edit

## 5) Sample XHTML
```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:ui="http://java.sun.com/jsf/facelets"
	xmlns:f="http://java.sun.com/jsf/core"
	xmlns:p="http://primefaces.org/ui"
	xmlns:h="http://java.sun.com/jsf/html">

<h:head>
	<title>PrimeFaces Test</title>
</h:head>
<h:body>

	<p:dataTable var="a" value="#{testView.list}" sortMode="multiple"
		editable="true" rowKey="#{a}">

		<p:column headerText="id" sortBy="#{a}">

			<p:outputLabel value="#{a}" />

		</p:column>

		<p:column style="width:32px" headerText="Editar">
			<p:rowEditor />
		</p:column>

	</p:dataTable>

</h:body>
</html>
```

## 6) Sample bean
```
@Named
@ViewScoped
public class TestView implements Serializable {
	private static final long serialVersionUID = 1L;

	List<String> list = Arrays.asList("sup1", "sup2", "sup3");

	public List<String> getList() {
		return list;
	}

	public void setList(List<String> list) {
		this.list = list;
	}

}

```
https://github.com/erickdeoliveiraleal/primefaces-test

**Comment by melloware** [comment]
@mertsincan I assume its related to this fix that was put in 7.0.2: https://github.com/primefaces/primefaces/issues/4557

**Comment by mertsincan** [comment]
Duplicate #4805
